unit-tests/Extension/ExceptionsTest.php

```diff
@@ -143,4 +144,37 @@ public function testExceptionRethrow()
             $this->assertSame(8, $e->getLine());
```

Comment:
https://github.com/phalcon/zephir/blob/master/test/exceptions.zep#L8 this is this eight line though as i see there in code maybe you are correct, it should be most likely https://github.com/phalcon/zephir/blob/master/test/exceptions.zep#L80 ?